muskathlon/views/website_muskathlon_view.xml

```diff
@@ -171,7 +171,7 @@
 
                                         </div>
                                         <div style="margin-top: 10px;">
-                                            <a t-attf-href="https://compassion.ch/parrainer-un-enfant/?eventId={{event.id}}&amp;registrationId={{registration.id}}" target="_blank" class="button button-large">Sponsor a child</a>
+                                            <a t-attf-href="https://compassion.ch/parrainer-un-enfant/?eventId={{event.id}}&amp;registrationId={{registration.id}}&amp;participantId={{participant.id}}&amp;participantName={{participant.name}}" target="_blank" class="button button-large">Sponsor a child</a>
```

Comment:
`https://compassion.ch/` can be replaced by the config `wordpress_host` present in odoo.conf file.
To achieve this I think the best way is to add a compute field in the registration object that will return the full URL.
This way we can redirect to test website on devel.